Fix remove double import from graphql/data in SwedishDetails

virrios opened the pull request:
## What?
Fix Removed double import from graphql/data in SwedishDetails

## Why?

Because someone had a quick trigger finger on merge..The linter should have caught this?

Commit:
Fix remove double import from graphql/data in SwedishDetails

styrelius reviewed (dismissed):
Yes, the linter should catch these types of things but apparently we don't have a rule for that? You're _more than welcome_ to add such a rule! 😃

Commit:
Updated Eslint to check for double imports and throw error

 src/client/data/graphql.tsx is a genereated file and has multiple
 duplicate imports so I added an overrride rule to the linting.
 I used the full path to the file and not *graphql.tsx because I think
 it should be strongly coupled with that particular file.

 You can add additional options for queryStrings etc but
 did not see it used. So keept it simple.

Commit:
Removed double import in Debugger noticed by the linter.

virrios commented:
![ESLINT](https://user-images.githubusercontent.com/19462389/100100076-89e29c80-2e60-11eb-9d24-96b43b75b9b5.gif)

styrelius reviewed (approved):
Wonderful! Ordning och reda pengar på freda osv 😀
